### ISSUE TITLE
What happened to the  version status table page that used to list all the current version supported?

### ISSUE THREAD

**Issue by fridayhut** [aws/aws-lambda-dotnet#806]
There used to be a  version status table page that listed all the .net core versions including minor patches that lambda supported.

I am specifically looking for the patch version of .net core 3.1.x that is currently used for lambda.

Thank you!

This is a :question: general question

**Comment by ashishdhingra** [comment]
Hi @fridayhut,

Good morning.

Thanks for posting guidance question. Let me try to find this information from development team.

Thanks,
Ashish

**Comment by normj** [comment]
Hi @fridayhut 

You are correct there used to be a version table in this repo's README. I took it out about a year or so ago because I couldn't consistently keep it up to date. This repo is for all of the .NET Lambda client tooling and although we do work closely with the Lambda service team especially for .NET related things we aren't part of that group that gets notifications when patches get rolled out to all of the regions. So the table was my attempt but always fell behind.

I originally added the table because at the start of .NET Core 2.0 or 2.1, sorry I can't remember, because if you built a project using a machine that had a later patch then the running environment .NET would fail to run. We added the table as well some checks in the client tooling to warn when this would happen but it often caused problems and we were happy to get rid of that once .NET fixed its issues. Since we don't have that concern anymore and I wasn't able to keep the table up to date I dropped the table.

Right now I don't plan on adding that back till I have a good way to automatic way to keep the table up to date. We are in the process of building yesterday's .NET patch releases and getting those pushed out to Lambda.

Norm
